app/models/ingredient.rb

```diff
@@ -1,3 +1,10 @@
 class Ingredient < ApplicationRecord
    validates_presence_of :name, :cost
+
+   has_many :recipe_ingredients
+   has_many :recipes, through: :recipe_ingredients
+
+   def self.sorted
+      order(:name)
+   end
```

Comment:
Good use of ActiveRecord! This method should have a unit test in the model_spec as well.